src/uct/rocm/base/rocm_base.c

```diff
@@ -287,10 +287,51 @@ ucs_status_t uct_rocm_base_detect_memory_type(uct_md_h md, const void *addr,
     return UCS_OK;
 }
 
+int uct_rocm_base_is_dmabuf_supported()
+{
+    int dmabuf_supported = 0;
+
+#if HAVE_HSA_AMD_PORTABLE_EXPORT_DMABUF == 1
```

Comment:
minor: for consistency with the other code:
```suggestion
#if HAVE_HSA_AMD_PORTABLE_EXPORT_DMABUF
```

Reply:
done, thanks!